pymc3/model.py

```diff
@@ -668,6 +669,13 @@ def __init__(
             self.deterministics = treelist()
             self.potentials = treelist()
 
+        from pymc3.printing import str_for_model
```

Comment:
why not move this to the global level?

Reply:
To avoid the circular import (`printing` depends on `model` and vice versa). I don't like the look of this either, but it seems to be the recommended way to avoid circular imports.